packages/guides/src/ReferenceResolvers/DocReferenceResolver.php

```diff
@@ -29,11 +34,27 @@ public function resolve(LinkInlineNode $node, RenderContext $renderContext): boo
             return false;
         }
 
-        $document = $renderContext->getProjectNode()->findDocumentEntry(
-            $this->documentNameResolver->canonicalUrl($renderContext->getDirName(), $node->getTargetReference()),
-        );
+        $canonicalDocumentName = $this->documentNameResolver->canonicalUrl($renderContext->getDirName(), $node->getTargetReference());
+
+        $document = $renderContext->getProjectNode()->findDocumentEntry($canonicalDocumentName);
         if ($document === null) {
-            return false;
+            $didYouMean = '';
+            $document = $renderContext->getProjectNode()->findProposedDocumentEntry($canonicalDocumentName);
+            if ($document !== null) {
+                $didYouMean = sprintf('Did you mean "%s"?', $document->getFile());
+            }
+
+            $this->logger->warning(
+                sprintf(
+                    'Document with name "%s" not found, required in file "%s". %s',
+                    $canonicalDocumentName,
+                    $renderContext->getCurrentFileName(),
+                    $didYouMean,
+                ),
+                array_merge($renderContext->getLoggerInformation(), $node->getDebugInformation()),
+            );
+
+            return true;
```

Comment:
Should this not be false? As there is nothing resolved. 

Maybe we should move the suggestions to a different resolver with the lowest priority, So the chain would be:

- DocReference
- Interlink
- ....
- SuggestionDocs
- SuggestionInterlink

Reply:
If we would do that we would output multiple warnings for the same reference?